engine/src/configuration/tests/config.py

```diff
@@ -0,0 +1,24 @@
+import yaml
```

Comment:
while YAML is a little easier to read; it's not built-in to Python so requires an additional package.
We're already using JSON elsewhere and JSON is built-in to Python. It might be wiser to be consistent with ourselves and just use what's built-in. The code below barely changes; the file format does a bit.

YAML is a little nicer looking; but for the moment ...not sure if it's the right solution.

Reply:
https://github.com/jbeder/yaml-cpp looks promising...but they removed their Boost dependency.
Boost currently lists YAML as a future development.